pages.de/linux/snap.md

```diff
@@ -0,0 +1,12 @@
+# snap
+
+> SNAP (Sentinel Application Platform) für die Prozessierung von Satellitendaten der Europäischen Raumfahrtagentur (ESA).
+> Download: <http://step.esa.int/main/>.
```

Comment:
`Weitere Informationen` is just a standard we have.
And the new link is a bit more specific, even if it's a download link.
```suggestion
> Weitere Informationen: <http://step.esa.int/main/download/snap-download/>.
```